src/RegistrationMsg.chpl

```diff
@@ -64,6 +64,10 @@ module RegistrationMsg
         return msgTuple;
     }
 
+    proc categorical_attachMsg(cmd: string, msgArgs: borrowed MessageArgs, st: borrowed SymTab): MsgTuple throws{
+
+    }
+
```

Comment:
i don't think is meant to be empty

Reply:
Oh man... That was left over from my initial thought that I would need a separate message. I forget the exact reason. It should actually be removed.